dropwizard-configuration/src/test/java/io/dropwizard/configuration/BaseConfigurationFactoryTest.java

```diff
@@ -427,21 +427,6 @@ public void throwsAnExceptionIfDefaultConfigurationCantBeInstantiated() throws E
                 "'io.dropwizard.configuration.BaseConfigurationFactoryTest.NonInsatiableExample'");
     }
 
-    @Test
-    public void printsDidYouMeanOnUnrecognizedField() throws Exception {
-        assertThatThrownBy(() -> factory.build(typoFile))
-            .isInstanceOf(ConfigurationParsingException.class)
-            .hasMessage(String.format("%s has an error:%n" +
-                "  * Unrecognized field at: propertis%n" +
-                "    Did you mean?:%n" +
-                "      - properties%n" +
-                "      - servers%n" +
-                "      - type%n" +
-                "      - name%n" +
-                "      - age%n" +
-                "        [2 more]%n", typoFile));
-    }
-
```

Comment:
Seems like this is redundant since `ConfigurationFactoryFactoryTest` [tests the same behavior](https://github.com/dropwizard/dropwizard/blob/0d143d7bc35351a135f8dc82c5e6f78d1eb39268/dropwizard-configuration/src/test/java/io/dropwizard/configuration/ConfigurationFactoryFactoryTest.java#L34-L44), but in a more apt place (since the behavior [is implemented by](https://github.com/dropwizard/dropwizard/blob/1e40fef4f4e8f048885b7ce6067bd9cd87a749b2/dropwizard-configuration/src/main/java/io/dropwizard/configuration/DefaultConfigurationFactoryFactory.java#L31) `DefaultConfigurationFactoryFactory`).